crates/torii/graphql/src/query/mod.rs

```diff
@@ -163,7 +175,17 @@ fn fetch_value(
                 row.try_get::<String, &str>(&column_name).map(Value::from)?,
             )),
         },
-        // fetch everything else as non-formated string
-        _ => Ok(row.try_get::<String, &str>(&column_name).map(Value::from)?),
+        // fetch everything else
+        _ => {
+            let mut value = row.try_get::<String, &str>(&column_name).map(Value::from)?;
+            // add timezone to naive datetime strings
+            if type_name == "DateTime" {
+                let dt_format = "\"%Y-%m-%d %H:%M:%S\"";
```

Comment:
Shouldn't these ideally be stored with the tz info? Are we basically assuming if it doesn't exist that it is always UTC?

Reply:
Found out that they're actually stored as UTC but the information was lost during the String conversion - I updated the `try_get::<String` to:
```rust
.try_get::<DateTime<Utc>, &str>(&column_name).to_rfc3339()
```
which gives the date string with the timezone information!
![image](https://github.com/dojoengine/dojo/assets/22559023/e38bba41-b325-428a-a33c-edb341b920ff)

Removed the formatting function then.